examples/resources/cloudavenue_/import.sh

```diff
@@ -0,0 +1,2 @@
+# use the name to import the resource
+terraform import cloudavenue_.example name
```

Comment:
An error .? the file name is incomplete, show path: "examples/resources/cloudavenue_"